### PR TITLE
pings: add README for local dev

### PR DESCRIPTION
**Pull request by unknwon**
## Test plan

CI

### CODE REVIEW

File: cmd/pings/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Pings service
+
+This directory contains source code for the [Pings service](https://handbook.sourcegraph.com/departments/engineering/teams/core-services/#pings-service). This document describes details for local development of the service.
```
Comment (unknwon) [inline cmd/pings/README.md:3]: This page will be added in a separate PR to handbook.

**Review by bobheadxi — APPROVED** [review]
nice! 😁
